ShareX/Forms/MainForm.cs

```diff
@@ -46,7 +46,6 @@ public partial class MainForm : HotkeyForm
         private bool forceClose, firstUpdateCheck = true;
         private UploadInfoManager uim;
         private ToolStripDropDownItem tsmiImageFileUploaders, tsmiTrayImageFileUploaders, tsmiTextFileUploaders, tsmiTrayTextFileUploaders;
-        private System.Threading.Timer updateTimer;
```

Comment:
This variable wasn't in use

On 20 Apr 2015 8:59 am, Jaex <notifications@github.com> wrote:

In ShareX/Forms/MainForm.cs<https://github.com/ShareX/ShareX/pull/636#discussion_r28669517>:

> @@ -46,7 +46,6 @@ public partial class MainForm : HotkeyForm
>          private bool forceClose, firstUpdateCheck = true;
>          private UploadInfoManager uim;
>          private ToolStripDropDownItem tsmiImageFileUploaders, tsmiTrayImageFileUploaders, tsmiTextFileUploaders, tsmiTrayTextFileUploaders;
> -        private System.Threading.Timer updateTimer;


Why you removed auto updating?

—
Reply to this email directly or view it on GitHub<https://github.com/ShareX/ShareX/pull/636/files#r28669517>.